14-05-2015/reverse-polish/reverse_polish.txt

```diff
@@ -0,0 +1,20 @@
+class ReversePolish
+  def self.evaluate(polish_str)
+    @res_arr = []
+    return 0 if polish_str.empty?
+    elements= polish_str.split(' ')
+    elements.each do |ele|
```

Comment:
`elements` is not used except in this line. You can skip to create local variable `elements`.